web/src/components/sign-up/sign-up.tsx

```diff
@@ -0,0 +1,88 @@
+import * as React from 'react';
```

Comment:
(Comment on the file name, not on this line):
Could we rename it to something that includes newsletter? I feel like sign-up might be confused with the authorization flow.

Reply:
Makes sense. Would you want to go with subscribe-newsletter instead?

Reply:
That sounds good to me!